iphone/Classes/UIModule.m

```diff
@@ -128,17 +128,29 @@ -(NSNumber*)RETURNKEY_CONTINUE
     return UIReturnKeyDefault;
 }
 
-MAKE_SYSTEM_PROP(KEYBOARD_DEFAULT,UIKeyboardTypeDefault);
-MAKE_SYSTEM_PROP(KEYBOARD_ASCII,UIKeyboardTypeASCIICapable);
-MAKE_SYSTEM_PROP(KEYBOARD_NUMBERS_PUNCTUATION,UIKeyboardTypeNumbersAndPunctuation);
-MAKE_SYSTEM_PROP(KEYBOARD_URL,UIKeyboardTypeURL);
-MAKE_SYSTEM_PROP(KEYBOARD_NUMBER_PAD,UIKeyboardTypeNumberPad);
-MAKE_SYSTEM_PROP(KEYBOARD_DECIMAL_PAD,UIKeyboardTypeDecimalPad);
-MAKE_SYSTEM_PROP(KEYBOARD_PHONE_PAD,UIKeyboardTypePhonePad);
-MAKE_SYSTEM_PROP(KEYBOARD_NAMEPHONE_PAD,UIKeyboardTypeNamePhonePad);
-MAKE_SYSTEM_PROP(KEYBOARD_EMAIL,UIKeyboardTypeEmailAddress);
-MAKE_SYSTEM_PROP(KEYBOARD_WEBSEARCH, UIKeyboardTypeWebSearch);
-MAKE_SYSTEM_PROP(KEYBOARD_TWITTER, UIKeyboardTypeTwitter);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_DEFAULT,UIKeyboardTypeDefault);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_URL,UIKeyboardTypeURL);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_ASCII,UIKeyboardTypeASCIICapable);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_NUMBERS_PUNCTUATION,UIKeyboardTypeNumbersAndPunctuation);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_NUMBER_PAD,UIKeyboardTypeNumberPad);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_DECIMAL_PAD,UIKeyboardTypeDecimalPad);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_PHONE_PAD,UIKeyboardTypePhonePad);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_NAMEPHONE_PAD,UIKeyboardTypeNamePhonePad);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_EMAIL,UIKeyboardTypeEmailAddress);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_WEBSEARCH, UIKeyboardTypeWebSearch);
+MAKE_SYSTEM_PROP(KEYBOARD_TYPE_TWITTER, UIKeyboardTypeTwitter);
+
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_DEFAULT,UIKeyboardTypeDefault, @"Titanium.UI.KEYBOARD_DEFAULT", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_DEFAULT");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_URL,UIKeyboardTypeURL, @"Titanium.UI.KEYBOARD_URL", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_URL");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_ASCII,UIKeyboardTypeASCIICapable, @"Titanium.UI.KEYBOARD_ASCII", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_ASCII");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_NUMBERS_PUNCTUATION,UIKeyboardTypeNumbersAndPunctuation, @"Titanium.UI.KEYBOARD_NUMBERS_PUNCTUATION", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_NUMBERS_PUNCTUATION");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_NUMBER_PAD,UIKeyboardTypeNumberPad, @"Titanium.UI.KEYBOARD_NUMBER_PAD", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_NUMBER_PAD");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_DECIMAL_PAD,UIKeyboardTypeDecimalPad, @"Titanium.UI.KEYBOARD_DECIMAL_PAD", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_DECIMAL_PAD");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(KEYBOARD_PHONE_PAD,UIKeyboardTypeNamePhonePad, @"Titanium.UI.KEYBOARD_PHONE_PAD", @"5.2.0", @"Titanium.UI.KEYBOARD_TYPE_PHONE_PAD");
```

Comment:
It is `UIKeyboardTypePhonePad`, not `UIKeyboardTypeNamePhonePad` for `KEYBOARD_PHONE_PAD`.